extension/observer/dockerobserver/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker Observer Extension
+
+**Status: beta**
+
+The `docker_observer` uses a Docker client and queries the Docker Engine API for running containers.
+
+Requires Docker API Version 1.22+.
+
+The collector will need permissions to access the Docker Engine API.
```

Comment:
Is there a docker doc that describes those required?

Reply:
I couldn't find a good doc but I can add a note that this means read access on the docker socket